schema.graphql

```diff
@@ -70,6 +75,35 @@ type Dataset {
   status: JobStatus
   inputPath: String
   uploadDateTime: String
+
+  access: AccessRights!
```

Comment:
We need to add group and projects here (access rights are going to be calculated based on those anyway).

Reply:
only projects as there is no direct connection between group and dataset